### ISSUE TITLE
feat(insights): backend updates to support pro/enterprise suggested tier

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#6721]
@avivapinchas @ackernaut @tghanken @gcrickman 
Do the definitions for each of the stats look correct? give the issue a thumbs up if so!

AC
- [ ] Box 1: `activeOrganizationCount` is defined an an organization with more than 1 user on it that has logged in within the last 30 days & at least 1 unarchived team
- [ ] Box 2: `activeTeamCount` is defined as a team within an active organization that has a currently active meeting or has completed a meeting within the last 30 days which
- [ ] Box3: `activeUserCount` is defined as a user on an active organization that has logged in within the last 30 days
- [ ] Box4: `meetingsCompleted` is defined as the number of meetings created by teams on organizations assigned to the domain
- [ ] `OrganizationUser.allowInsights` is replaced with `Organizationuser.suggestedInsightsTier`. the mutation to set it is updated with that tier enum e.g. `toggleInsights(orgId: "foo", emails: ["bob@foo.co"], tier: enterprise)`. if tier is null, the value is unset. 

Estimate: 5 hours

**Comment by tghanken** [comment]
The only one I'm not sure about is `activeTeamCount`.  There's a fair amount of meetings that are just never closed which would inflate this number based on the current definition.  I'd like to replace "has a currently active meeting" with something like "has had user activity on a meeting".  Could we just check against the `updatedAt` field for this?

**Comment by mattkrick** [comment]
good call. will make sure updatedAt is > 30 days ago?

**Comment by ackernaut** [comment]
For Box 2, do we want this to be **active teams** or **total teams created**?

For Box 3, do we want this to be **active users** or **all registered users**?

For [the PDF report](https://www.notion.so/parabol/Domain-stats-discovery-957a5042781645cb92c4df30ce3e3ef6#7cc5618384404931a558dc0a63849a5b), I believe it’s total teams and registered users. If we were going to compare performance of the in-app usage stats with the PDF by [some A/B sales experiment](https://parabol.slack.com/archives/C03HPHWH55W/p1654887662447339?thread_ts=1654119550.015139&cid=C03HPHWH55W), should the information be close to the same?

cc @gcrickman

**Comment by avivapinchas** [comment]
> Box 2: activeTeamCount is defined as a team within an active organization that has a currently active meeting or has completed a meeting within the last 30 days

I would exclude single-member teams from this - some folks have personal teams or personal orgs, which aren't really representative of the company's usage 

>  I believe it’s total teams and registered users. 

🤔 So sticking with what's in the PDF would give us a more apples-to-apples comparison, _but_ I think 'active' makes a better case for value 

They ultimately pay based on active, so that feels like a more relevant metric to show, imo.

**Comment by mattkrick** [comment]
Definitions updated! let me know if the new ones look good.

> Maybe we count the number of meetings started, and the label is just "meetings"?

oops, i meant `meetingCount`, friday brain 🙈. that's the same metric that we send to hubspot today: sales_op_meeting_count

**Comment by ackernaut** [comment]
LGTM

**Comment by avivapinchas** [comment]
One nit: Box2 should also include the 2-team member condition: _has at least 2 team members_ 

Running meetings on a team by yourself is probably a bit of an edge case, but I could see folks doing it to test out the tool, and it's a behavior I believe we're considering encouraging in onboarding, so they can get more confident with how parabol works

**Comment by ackernaut** [comment]
In [these Figma views](https://www.figma.com/file/454u4uRYZlZMtmUtgQAQ3G/Usage-Stats-v1?node-id=139%3A8244) I updated the labels:
- **Workspace**(**s**) (no need to confuse by including ‘active’ here)
- **Active Teams**
- **Active Members**
- **Meetings** (started)

Feedback on those labels?

Should Active Members be Active Seats? Which of the 2 might be more normalized?

**Comment by mattkrick** [comment]
> One nit: Box2 should also include the 2-team member condition: has at least 2 team members

@avivapinchas updated. assuming both team members must have logged in within the last 30 days?

> Workspace(s) (no need to confuse by including ‘active’ here)

@ackernaut this'll be the only place in the app where we use the word "workspace". Could we keep the word "Organization" and then in a subsequent sprint rename "Organization" to "Workspace" across the whole app?

**Comment by ackernaut** [comment]
@mattkrick Yes! We can use Organization(s) until we do the work to update the naming everywhere. Sorry for missing that detail, I like to live in the future, at least slightly :)

**Comment by avivapinchas** [comment]
> assuming both team members must have logged in within the last 30 days?
I suppose so

**Comment by ackernaut** [comment]
I agree _members_ feels a bit better than _seats_.
